src/App.js

```diff
@@ -2,21 +2,32 @@ import './App.css';
 import React from 'react';
 
 function App() {
-  const [textInput, setTextInput] = React.useState('Here is some example text.');
+  const [textInput, setTextInput] = React.useState(
+    'Here is some example text.'
+  );
   const [conversionMode, setConversionMode] = React.useState('lowercase');
   const [textOutput, setTextOutput] = React.useState('');
+  const [errorMessage, setErrorMessage] = React.useState('');
 
   const handleRadioChange = event => {
     setConversionMode(event.target.value);
-  }
+  };
 
   const handleTextareaChange = event => {
     setTextInput(event.target.value);
   };
 
   const handleSubmit = event => {
     event.preventDefault();
-    setTextOutput('Your formatted text will go here!')
+    setErrorMessage('');
+    if (!textInput) {
+      setErrorMessage('You must input text first!');
+      return;
+    }
+    conversionMode === 'lowercase'
+      ? setTextOutput(textInput.toLowerCase())
+      : setTextOutput(textInput.toUpperCase());
+    setTextInput('');
```

Comment:
Clearing the text input could be a usability issue. What if I made a mistake and want to edit the original input? Or try the same input with both the upper and lower case?